components/inputs/test/input-date.test.js

```diff
@@ -55,7 +55,7 @@ describe('d2l-input-date', () => {
 			await dropdownContent.updateComplete;
 			dropdown.toggleOpen();
 			await oneEvent(dropdown, 'd2l-dropdown-open');
-			const focusTrap = getChildElem(elem, 'd2l-focus-trap');
+			const focusTrap = getChildElem(dropdownContent, 'd2l-focus-trap');
```

Comment:
Could probably just delete these two tests since technically this behaviour now belongs to dropdown.